catroid/src/main/java/org/catrobat/catroid/devices/arduino/ArduinoImpl.java

```diff
@@ -256,7 +245,7 @@ public static int getIndexOfPinOnPort(int pin) {
 
 	private void sendAnalogFirmataMessage(int pin, int value) {
 		sendFirmataMessage(new SetPinModeMessage(pin, SetPinModeMessage.PIN_MODE.PWM.getMode()));
-		sendFirmataMessage(new AnalogMessage(pin, castValue(value)));
+		sendFirmataMessage(new AnalogMessage(pin, value));
```

Comment:
does this change the behaviour of existing programs?

Reply:
Yes! (`Behavior Change` label)

Previously values were clamped to [0-100] and then mapped to [0-255].
Now values are just passed to Firmata and Arduino.

So the language version needs to be increased for the next release, I guess.

Reply:
Updates?
This is a really important change for the Arduino feature. The sooner it's merged, the better :wink:.